### PR TITLE
Starknet-Foundry

### PR DESCRIPTION
**Pull request by malakbenh**

### CODE REVIEW

**Comment by malakbenh** [comment]
Starknet-Foundry-commit

File: course/8-Get-Started-with-starkent-foundry.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<div dir="rtl">
```
Comment (KhairallahA) [inline course/8-Get-Started-with-starkent-foundry.md:1]: i don't think we need to add this line, you can delete it.

File: course/8-Get-Started-with-starkent-foundry.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+   سيؤدي ذلك إلى إنشاء دليل `target`.
+3. انقل دليل `target` إلى الموقع الذي تريده، مثل:
+   ```
+   ~/.starknet-foundry
+   ```
+4. تأكد من إضافة `DESIRED_LOCATION/target/release/` إلى متغير البيئة `PATH`.
+
+
+
+</div>
```
Comment (KhairallahA) [inline course/8-Get-Started-with-starkent-foundry.md:94]: same

**Review by KhairallahA — CHANGES_REQUESTED** [review]
great work!

**Comment by KhairallahA** [comment]
@malakbenh merged!

**Pull request merged**